packages/blade/src/components/Card/CardInteractive.stories.tsx

```diff
@@ -0,0 +1,709 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable no-alert */
+import type { Meta } from '@storybook/react';
+import React from 'react';
+import type { CardProps } from './Card';
+import {
+  CardBody,
+  Card,
+  CardFooter,
+  CardFooterLeading,
+  CardFooterTrailing,
+  CardHeader,
+  CardHeaderLeading,
+  CardHeaderTrailing,
+  CardHeaderIcon,
+  CardHeaderCounter,
+  CardHeaderBadge,
+} from '.';
+import { Code, Text, Title } from '~components/Typography';
+import { RupeeIcon } from '~components/Icons';
+import { Link } from '~components/Link';
+import { Box } from '~components/Box';
+import { Button } from '~components/Button';
+import { VisuallyHidden } from '~components/VisuallyHidden';
+import { Amount } from '~components/Amount';
+import { isReactNative } from '~utils';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import { Sandbox } from '~utils/storybook/Sandbox';
+
+const Page = (): React.ReactElement => {
+  return (
+    <StoryPageWrapper
+      componentName="Interactive Card"
+      componentDescription="Enhancing the Card component to add additional interactions and behaviour. This includes making the card clickable, hoverable, linkable, selectable and more."
+      figmaURL={{
+        paymentTheme:
+          'https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?node-id=21248%3A400833&t=ZCWT255jVK78xf1J-4',
+        bankingTheme:
+          'https://www.figma.com/file/sAdplk2uYnI2ILnDKUxycW/Blade---Banking-Dark?node-id=12791%3A336279&t=ZCWT255jVK78xf1J-4',
+      }}
+    >
+      <Title>Usage</Title>
+      <Sandbox editorHeight={500}>
+        {`
+        import React from 'react';
+        import { Card, CardBody, Box, Text, Amount, VisuallyHidden } from '@razorpay/blade/components';
+
+        type HiddenInputProps = {
+          onChange: (value: string) => void;
+          value: string;
+          name: string;
+          type?: string;
+        }
+        const HiddenInput = ({
+          onChange,
+          value,
+          name,
+          type,
+        }: HiddenInputProps): React.ReactElement => {
+          return (
+            <VisuallyHidden>
+              <input
+                type={type ?? 'radio'}
+                onChange={(e) => {
+                  onChange(e.target.value);
+                }}
+                name={name}
+                value={value}
+              />
+            </VisuallyHidden>
+          );
+        };
+
+        const App = (): React.ReactElement => {
+          const [selected, setSelected] = React.useState('free');
+
+          return (
+            <Box display="flex" flexWrap="wrap" gap="spacing.5">
+              <Card
```

Comment:
So we have focus ring when we have onClick but not when we have isSelected? if we talk about checkbox/radio all of them have focus ring right? so shouldn't the card with selection also have focus ring?

Reply:
isSelected and focus are two different states. 

isSelected is purely a visual indicator while on the other hand focus is just normal browser focus. Just because a Card is in `selected` state doesn't mean it will automatically also be `focused`

Reply:
oh right